collector/src/main/java/com/bitmovin/analytics/adapters/ExoPlayerAdapter.java

```diff
@@ -82,7 +83,11 @@ private long getPosition() {
             Timeline.Period firstPeriodInWindow = new Timeline.Period();
             if (firstPeriodInWindowIndex >= 0 && firstPeriodInWindowIndex < timeline.getPeriodCount()) {
                 timeline.getPeriod(firstPeriodInWindowIndex, firstPeriodInWindow);
-                return (exoplayer.getCurrentPosition() - firstPeriodInWindow.getPositionInWindowMs()) / 1000;
+                long position = (exoplayer.getCurrentPosition() - firstPeriodInWindow.getPositionInWindowMs());
+                if (position < 0) {
```

Comment:
Any clue why the calculated position can be negative?

Reply:
I am not sure. I debugged some more this morning and can not find a pattern. Most of the time it doesnt happen, but when it does it is very small values that use to get rounded to 0 when divided by 0. Potentially an exoplayer issue?